src/quadbez.rs

```diff
@@ -265,6 +280,11 @@ impl ParamCurveArclen for QuadBez {
         let c2 = 2.0 * c.sqrt();
         let ba_c2 = b * a2 + c2;
 
+        if a2.is_infinite() {
+            // The arclength is zero
```

Comment:
This is the case where all control points are on top of each other? Would it be addressed by changing `a < 5e-4 * c` to `a <= 5e-4 * c` above? Also it would be great for this case to be tested.

Reply:
`a2` is purely calculated from the value of `a`, which means changing the code to `a <= 5e-4 * c` doesn't necessarily fix the issue. Also, the difference between < and <= is infinitesimally small and therefore doesn't really matter with real values. Since it's existing code that I didn't write, I don't want to change < to <= without more discussion since I don't entirely know the intent there.

Also, I'll add a test in my next commit.